WooCommerce/Classes/View Modifiers/WooStyleModifiers.swift

```diff
@@ -120,4 +127,8 @@ extension View {
     func errorStyle() -> some View {
         self.modifier(ErrorStyle())
     }
+
+    func navigationBarStyle() -> some View {
+        self.modifier(NavigationBarStyle())
+    }
```

Comment:
[Q] Do you think we should we add this extension only for `NavigationView`? Or is it useful for any view?

Also probably something like `wooNavigationBarStyle` or `accentNavigationBarStyle` could be more describing, what do you think?

Reply:
> [Q] Do you think we should we add this extension only for NavigationView? Or is it useful for any view?

We have some SwiftUI views (e.g. `ApplicationLogDetailView`, `EditOrderAddressForm`) that need the modifier but don't have a `NavigationView`, so I think we need this extension for any view.

> Also probably something like wooNavigationBarStyle or accentNavigationBarStyle could be more describing, what do you think?

I think `wooNavigationBarStyle` makes a lot of sense here, thanks! I updated the naming in dcf2e04.